2023-05-24 00:06:43 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 00:12:43 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 00:27:43 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 01:15:14 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 07:06:38 WARN [com.zaxxer.hikari.pool.HikariPool]: HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=5h39m53s517ms8Âµs900ns).
2023-05-24 07:06:51 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 07:06:51 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 07:13:35 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 07:13:35 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 07:15:26 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 07:15:26 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 07:37:50 WARN [com.zaxxer.hikari.pool.HikariPool]: HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=11m10s227ms912Âµs300ns).
2023-05-24 07:46:03 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 07:46:09 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 07:46:09 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 07:50:23 WARN [com.zaxxer.hikari.pool.HikariPool]: HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=4m2s100ms38Âµs300ns).
2023-05-24 07:53:29 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 07:57:13 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 07:57:13 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 07:58:30 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 07:58:30 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 07:58:57 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 07:58:57 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 07:59:33 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 07:59:33 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 08:00:36 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 08:00:36 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 08:01:40 ERROR [com.example.yin.exception.GlobalExceptionHandler]: ãç³»ç»å¼å¸¸ãNo converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8'
org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8'
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:312)
	at org.springframework.web.servlet.mvc.method.annotation.HttpEntityMethodProcessor.handleReturnValue(HttpEntityMethodProcessor.java:219)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:808)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1067)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:655)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:711)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:578)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:517)
	at org.apache.catalina.core.StandardHostValve.custom(StandardHostValve.java:384)
	at org.apache.catalina.core.StandardHostValve.status(StandardHostValve.java:249)
	at org.apache.catalina.core.StandardHostValve.throwable(StandardHostValve.java:344)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1732)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
2023-05-24 08:01:40 WARN [org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver]: Failure in @ExceptionHandler com.example.yin.exception.GlobalExceptionHandler#exception(Exception)
org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class com.example.yin.common.R] with preset Content-Type 'application/force-download;charset=UTF-8'
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:312)
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor.handleReturnValue(RequestResponseBodyMethodProcessor.java:183)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135)
	at org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver.doResolveHandlerMethodException(ExceptionHandlerExceptionResolver.java:428)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodExceptionResolver.doResolveException(AbstractHandlerMethodExceptionResolver.java:75)
	at org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver.resolveException(AbstractHandlerExceptionResolver.java:142)
	at org.springframework.web.servlet.handler.HandlerExceptionResolverComposite.resolveException(HandlerExceptionResolverComposite.java:80)
	at org.springframework.web.servlet.DispatcherServlet.processHandlerException(DispatcherServlet.java:1327)
	at org.springframework.web.servlet.DispatcherServlet.processDispatchResult(DispatcherServlet.java:1138)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1084)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:655)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:711)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:578)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:517)
	at org.apache.catalina.core.StandardHostValve.custom(StandardHostValve.java:384)
	at org.apache.catalina.core.StandardHostValve.status(StandardHostValve.java:249)
	at org.apache.catalina.core.StandardHostValve.throwable(StandardHostValve.java:344)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1732)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
2023-05-24 08:01:40 WARN [org.springframework.web.servlet.mvc.support.DefaultHandlerExceptionResolver]: Resolved [org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8']
2023-05-24 08:01:54 ERROR [com.example.yin.exception.GlobalExceptionHandler]: ãç³»ç»å¼å¸¸ãNo converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8'
org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8'
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:312)
	at org.springframework.web.servlet.mvc.method.annotation.HttpEntityMethodProcessor.handleReturnValue(HttpEntityMethodProcessor.java:219)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:808)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1067)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:655)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:711)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:578)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:517)
	at org.apache.catalina.core.StandardHostValve.custom(StandardHostValve.java:384)
	at org.apache.catalina.core.StandardHostValve.status(StandardHostValve.java:249)
	at org.apache.catalina.core.StandardHostValve.throwable(StandardHostValve.java:344)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1732)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
2023-05-24 08:01:54 WARN [org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver]: Failure in @ExceptionHandler com.example.yin.exception.GlobalExceptionHandler#exception(Exception)
org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class com.example.yin.common.R] with preset Content-Type 'application/force-download;charset=UTF-8'
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:312)
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor.handleReturnValue(RequestResponseBodyMethodProcessor.java:183)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135)
	at org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver.doResolveHandlerMethodException(ExceptionHandlerExceptionResolver.java:428)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodExceptionResolver.doResolveException(AbstractHandlerMethodExceptionResolver.java:75)
	at org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver.resolveException(AbstractHandlerExceptionResolver.java:142)
	at org.springframework.web.servlet.handler.HandlerExceptionResolverComposite.resolveException(HandlerExceptionResolverComposite.java:80)
	at org.springframework.web.servlet.DispatcherServlet.processHandlerException(DispatcherServlet.java:1327)
	at org.springframework.web.servlet.DispatcherServlet.processDispatchResult(DispatcherServlet.java:1138)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1084)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:655)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:711)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:578)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:517)
	at org.apache.catalina.core.StandardHostValve.custom(StandardHostValve.java:384)
	at org.apache.catalina.core.StandardHostValve.status(StandardHostValve.java:249)
	at org.apache.catalina.core.StandardHostValve.throwable(StandardHostValve.java:344)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1732)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
2023-05-24 08:01:54 WARN [org.springframework.web.servlet.mvc.support.DefaultHandlerExceptionResolver]: Resolved [org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8']
2023-05-24 08:32:30 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 08:33:00 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 08:33:00 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 09:02:31 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 09:02:31 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 09:38:32 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 17:47:07 INFO [org.hibernate.validator.internal.util.Version]: HV000001: Hibernate Validator 5.2.4.Final
2023-05-24 17:47:07 INFO [com.example.yin.YinMusicApplication]: Starting YinMusicApplication using Java 1.8.0_371 on taffy with PID 3508 (C:\SnowChar\work5\tmp\music-server-5-23-15h\target\classes started by 97373 in C:\SnowChar\work5\tmp\music-server-5-23-15h)
2023-05-24 17:47:07 INFO [com.example.yin.YinMusicApplication]: The following profiles are active: dev
2023-05-24 17:47:07 INFO [org.springframework.boot.devtools.env.DevToolsPropertyDefaultsPostProcessor]: Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2023-05-24 17:47:07 INFO [org.springframework.boot.devtools.env.DevToolsPropertyDefaultsPostProcessor]: For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2023-05-24 17:47:10 INFO [com.example.yin.config.JedisConfig]: redisè¿æ¥æå-127.0.0.1:6379
2023-05-24 17:47:10 INFO [org.springframework.boot.web.embedded.tomcat.TomcatWebServer]: Tomcat initialized with port(s): 8888 (http)
2023-05-24 17:47:10 INFO [org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext]: Root WebApplicationContext: initialization completed in 3108 ms
2023-05-24 17:47:11 INFO [org.springframework.boot.devtools.autoconfigure.OptionalLiveReloadServer]: LiveReload server is running on port 35729
2023-05-24 17:47:11 INFO [org.springframework.boot.web.embedded.tomcat.TomcatWebServer]: Tomcat started on port(s): 8888 (http) with context path ''
2023-05-24 17:47:12 INFO [com.example.yin.YinMusicApplication]: Started YinMusicApplication in 4.873 seconds (JVM running for 5.892)
2023-05-24 17:47:21 INFO [org.springframework.web.servlet.DispatcherServlet]: Initializing Servlet 'dispatcherServlet'
2023-05-24 17:47:21 INFO [org.springframework.web.servlet.DispatcherServlet]: Completed initialization in 3 ms
2023-05-24 17:47:24 INFO [com.zaxxer.hikari.HikariDataSource]: HikariPool-1 - Starting...
2023-05-24 17:47:24 INFO [com.zaxxer.hikari.HikariDataSource]: HikariPool-1 - Start completed.
2023-05-24 17:47:46 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 17:47:46 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 17:48:50 INFO [com.example.yin.service.impl.RecommendServiceImpl]: å½åç¨æ·æªç»å½ï¼éç¨éæºæ¨èç®æ³æ¨è8é¦ï¼æµè¡åº¦æ¨èç®æ³æ¨è2é¦
2023-05-24 17:50:41 INFO [com.example.yin.service.impl.RecommendServiceImpl]: å½åç¨æ·æªç»å½ï¼éç¨éæºæ¨èç®æ³æ¨è8é¦ï¼æµè¡åº¦æ¨èç®æ³æ¨è2é¦
2023-05-24 17:50:47 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=asdasde
2023-05-24 17:50:47 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=asdasde
2023-05-24 17:55:38 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=asdasde
2023-05-24 17:55:38 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=asdasde
2023-05-24 18:06:41 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=asdasde
2023-05-24 18:06:41 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=asdasde
2023-05-24 18:08:15 INFO [com.example.yin.service.impl.RecommendServiceImpl]: å½åç¨æ·å·²ç»å½ï¼éç¨æ··åæ¨èç®æ³
2023-05-24 18:08:47 INFO [com.example.yin.service.impl.RecommendServiceImpl]: å½åç¨æ·å·²ç»å½ï¼éç¨æ··åæ¨èç®æ³
2023-05-24 18:09:17 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 18:09:17 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 18:10:21 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=asdasde
2023-05-24 18:10:21 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=asdasde
2023-05-24 18:10:41 ERROR [com.example.yin.exception.GlobalExceptionHandler]: ãç³»ç»å¼å¸¸ãNo converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8'
org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8'
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:312)
	at org.springframework.web.servlet.mvc.method.annotation.HttpEntityMethodProcessor.handleReturnValue(HttpEntityMethodProcessor.java:219)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:808)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1067)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:655)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:711)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:578)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:517)
	at org.apache.catalina.core.StandardHostValve.custom(StandardHostValve.java:384)
	at org.apache.catalina.core.StandardHostValve.status(StandardHostValve.java:249)
	at org.apache.catalina.core.StandardHostValve.throwable(StandardHostValve.java:344)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1732)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
2023-05-24 18:10:41 WARN [org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver]: Failure in @ExceptionHandler com.example.yin.exception.GlobalExceptionHandler#exception(Exception)
org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class com.example.yin.common.R] with preset Content-Type 'application/force-download;charset=UTF-8'
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:312)
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor.handleReturnValue(RequestResponseBodyMethodProcessor.java:183)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135)
	at org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver.doResolveHandlerMethodException(ExceptionHandlerExceptionResolver.java:428)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodExceptionResolver.doResolveException(AbstractHandlerMethodExceptionResolver.java:75)
	at org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver.resolveException(AbstractHandlerExceptionResolver.java:142)
	at org.springframework.web.servlet.handler.HandlerExceptionResolverComposite.resolveException(HandlerExceptionResolverComposite.java:80)
	at org.springframework.web.servlet.DispatcherServlet.processHandlerException(DispatcherServlet.java:1327)
	at org.springframework.web.servlet.DispatcherServlet.processDispatchResult(DispatcherServlet.java:1138)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1084)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:655)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:711)
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:578)
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:517)
	at org.apache.catalina.core.StandardHostValve.custom(StandardHostValve.java:384)
	at org.apache.catalina.core.StandardHostValve.status(StandardHostValve.java:249)
	at org.apache.catalina.core.StandardHostValve.throwable(StandardHostValve.java:344)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1732)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
2023-05-24 18:10:41 WARN [org.springframework.web.servlet.mvc.support.DefaultHandlerExceptionResolver]: Resolved [org.springframework.http.converter.HttpMessageNotWritableException: No converter for [class java.util.LinkedHashMap] with preset Content-Type 'application/force-download;charset=UTF-8']
2023-05-24 18:14:00 INFO [com.example.yin.service.impl.RecommendServiceImpl]: å½åç¨æ·æªç»å½ï¼éç¨éæºæ¨èç®æ³æ¨è8é¦ï¼æµè¡åº¦æ¨èç®æ³æ¨è2é¦
2023-05-24 18:15:02 INFO [com.example.yin.service.impl.RecommendServiceImpl]: å½åç¨æ·æªç»å½ï¼éç¨éæºæ¨èç®æ³æ¨è8é¦ï¼æµè¡åº¦æ¨èç®æ³æ¨è2é¦
2023-05-24 18:15:18 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=123456
2023-05-24 18:15:18 INFO [com.example.yin.annotation.validated.RegularRuleValidate]: æ°æ®æ ¡éªå¥åargs=snowchar
2023-05-24 18:26:41 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 18:26:41 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 18:40:11 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 18:47:11 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 18:47:11 WARN [redis.clients.jedis.JedisFactory]: Error while close
redis.clients.jedis.exceptions.JedisException: Could not return the broken resource to the pool
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:126)
	at redis.clients.jedis.util.Pool.returnBrokenResource(Pool.java:103)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:382)
	at redis.clients.jedis.JedisPool.returnResource(JedisPool.java:15)
	at redis.clients.jedis.Jedis.close(Jedis.java:4067)
	at redis.clients.jedis.JedisFactory.destroyObject(JedisFactory.java:166)
	at org.apache.commons.pool2.PooledObjectFactory.destroyObject(PooledObjectFactory.java:127)
	at org.apache.commons.pool2.impl.GenericObjectPool.destroy(GenericObjectPool.java:611)
	at org.apache.commons.pool2.impl.GenericObjectPool.evict(GenericObjectPool.java:729)
	at org.apache.commons.pool2.impl.BaseGenericObjectPool$Evictor.run(BaseGenericObjectPool.java:160)
	at org.apache.commons.pool2.impl.EvictionTimer$WeakRunner.run(EvictionTimer.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.IllegalStateException: Invalidated object not currently part of this pool
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:936)
	at org.apache.commons.pool2.impl.GenericObjectPool.invalidateObject(GenericObjectPool.java:914)
	at redis.clients.jedis.util.Pool.returnBrokenResourceObject(Pool.java:124)
	... 17 more
2023-05-24 21:03:34 WARN [com.zaxxer.hikari.pool.HikariPool]: HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=1h56m8s210ms533Âµs500ns).
2023-05-24 22:28:33 WARN [com.zaxxer.hikari.pool.HikariPool]: HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=57m57s936ms217Âµs399ns).
2023-05-24 23:33:27 WARN [com.zaxxer.hikari.pool.HikariPool]: HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=29m53s4ms866Âµs300ns).
2023-05-24 23:36:31 INFO [com.zaxxer.hikari.HikariDataSource]: HikariPool-1 - Shutdown initiated...
2023-05-24 23:36:31 INFO [com.zaxxer.hikari.HikariDataSource]: HikariPool-1 - Shutdown completed.
